content/tutorials/get-started/get-started-with-the-rippled-api.md

```diff
@@ -224,3 +102,8 @@ The response format for commandline methods is the same as JSON-RPC responses, b
 - **References:**
     - [rippled API Reference](rippled-api.html)
     - [Ripple Data API v2](data-api.html)
+
+<!--{# common link defs #}-->
+{% include '_snippets/rippled-api-links.md' %}			
+{% include '_snippets/tx-type-links.md' %}			
+{% include '_snippets/rippled_versions.md' %}
```

Comment:
Seeing this:
![image](https://user-images.githubusercontent.com/354723/78080563-7f790d00-737c-11ea-824c-f1c5aeb511f2.png)

Reply:
Yes, that won't show up when it's built with [Dactyl](https://github.com/ripple/dactyl/).

Reply:
Great. Thx!